llvm/lib/Transforms/Vectorize/VPlanUtils.h

```diff
@@ -97,6 +97,14 @@ bool isUniformAcrossVFsAndUFs(VPValue *V);
 /// Returns the header block of the first, top-level loop, or null if none
 /// exist.
 VPBasicBlock *getFirstLoopHeader(VPlan &Plan, VPDominatorTree &VPDT);
+
+/// Returns the VPValue representing the uncounted exit comparison if all the
+/// recipes needed to form the condition within the vector loop body were
+/// matched.
```

Comment:
> [...] if all the recipes needed to form the condition within the vector loop body were matched.

This isn't really saying much, do you mean to say:

> Returns the VPValue representing the uncounted exit comparison for a restricted set of conditions that are considered safe to calculate for the next vector iteration.

? If so, then I'd rather you write that explicitly.